frontend/src/components/PreviewReview/PreviewReview.tsx

```diff
@@ -0,0 +1,174 @@
+import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  HStack,
+  Icon,
+  Image,
+  Stack,
+  Tag,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { ReviewResponse } from "../../types/ReviewTypes";
+
+/**
+ * The model defining the props for the Preview Review component
+ */
+interface PreviewReviewProps {
+  /** The review to be previewed */
+  review: ReviewResponse;
+}
+
+/**
+ * This component allows an admin to preview their review and get an understanding of
+ * what the subscribers of the review will see
+ */
+const PreviewReview = ({ review }: PreviewReviewProps): React.ReactElement => {
+  const [currentBook, setCurrentBook] = useState<number>(0);
+
+  const handleLeftClick = () => {
```

Comment:
How difficult do you think it would be to add a brief transition/animation here? It could be as simple as a fade in/out if not a scroll. 

If you think it's too much work, we can make it a separate ticket

Reply:
Briefly looked into it - I think it would be better as a separate ticket

Reply:
Cool, thanks for looking